kernels/openvx-core/host/vx_accumulate_square_host.c

```diff
@@ -297,7 +297,7 @@ vx_status tivxAddKernelAccumulateSquare(vx_context context)
         {
             status = vxAddParameterToKernel(kernel,
                         index,
-                        (vx_enum)VX_OUTPUT,
+                        (vx_enum)VX_BIDIRECTIONAL,
```

Comment:
This change is modifying the existing accumulate square kernel but the spec extension defines new kernels.